docs: operators and clusters to watch (#6986)
strimzi/strimzi-kafka-operator: documentation/modules/deploying/proc-deploy-topic-operator-with-cluster-operator.adoc

This procedure describes how to deploy the Topic Operator using the Cluster Operator.

You configure the `entityOperator` property of the `Kafka` resource to include the `topicOperator`.
By default, the Topic Operator watches for `KafkaTopic` resources in the namespace of the Kafka cluster deployed by the Cluster Operator.
You can also specify a namespace using `watchedNamespace` in the Topic Operator `spec`.

> The Topic Operator can only watch a single Kafka cluster in a namespace.
> If you want to use the operator with more than one Kafka cluster, install the clusters to separate namespaces.

Reviewer: ```suggestion
The Topic Operator can only watch a single namespace.
And a single namespace should be watched only by a single Topic Operator.
```

Author: Can we just say: _A single Topic Operator can watch a single namespace._

This procedure describes how to deploy the Topic Operator using the Cluster Operator.

You configure the `entityOperator` property of the `Kafka` resource to include the `topicOperator`.
By default, the Topic Operator watches for `KafkaTopic` resources in the namespace of the Kafka cluster deployed by the Cluster Operator.
You can also specify a namespace using `watchedNamespace` in the Topic Operator `spec`.

> A single Topic Operator can watch a single namespace. 

Reviewer: ```suggestion
A single Topic Operator can watch a single namespace.
A single namespace should be watched only by one Topic Operator.

If you use Strimzi to deploy multiple Kafka clusters into the same namespace, you should either disable the Topic Operator for all but one Kafka clusters or you should use the `watchedNamespace` option to configure them to watch other namespaces.
```
